packages/commonwealth/client/scripts/views/components/comments/edit_comment.tsx

```diff
@@ -27,21 +27,19 @@ export const EditComment = (props: EditCommentProps) => {
     updatedCommentsCallback,
   } = props;
 
-  const [quillEditorState, setQuillEditorState] = React.useState<QuillEditor>();
+  const body = shouldRestoreEdits && savedEdits ? savedEdits : comment.text;
+
+  const [contentDelta, setContentDelta] = React.useState<DeltaStatic>(createDeltaFromText(body));
   const [saving, setSaving] = React.useState<boolean>();
 
-  const body = shouldRestoreEdits && savedEdits ? savedEdits : comment.text;
+  const editorValue = getTextFromDelta(contentDelta);
 
   return (
     <div className="EditComment">
-      <QuillEditorComponent
-        contentsDoc={body}
-        oncreateBind={(state: QuillEditor) => {
-          setQuillEditorState(state);
-        }}
-        imageUploader
-        theme="snow"
-        editorNamespace={`edit-comment-${comment.id}`}
+      <ReactQuillEditor
+        theme='snow'
```

Comment:
theme should be defined in global react quill component so will be the same for all instances